src/routes/prof-nidurstodur/KosningaProf.js

```diff
@@ -19,9 +19,15 @@ const initialAnswers = questions =>
   }, {});
 
 const marks = {
-  1: 'Mjög ósammála',
+  0: 'Mjög ósammála',
   3: 'Hlutlaus',
-  5: 'Mjög sammála',
+  4: 'Mjög sammála',
+};
+
+const IMPORTANT_SYMBOL = '!';
+
+const isImportant = answer => {
+  return answer.includes(IMPORTANT_SYMBOL);
 };
```

Comment:
Maybe this is just:

```suggestion
const isImportant = answer => {
  return answer.includes('!');
};
```

?